src/plugins/intel_cpu/tests/functional/shared_tests_instances/snippets/utils.hpp

```diff
@@ -0,0 +1,34 @@
+// Copyright (C) 2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "internal_properties.hpp"
+#include "utils/cpu_test_utils.hpp"
+#include "openvino/runtime/system_conf.hpp"
+
+namespace ov {
+namespace test {
+namespace snippets {
+
+#define SNIPPETS_TESTS_STATIC_SHAPES(...) static_shapes_to_test_representation(std::vector<std::vector<ov::Shape>>{__VA_ARGS__})
```

Comment:
Do we really need `SNIPPETS_` in the name? It's already in the `ov::test::snippets` namespace.

Reply:
As far as I know, `macros` are not defined in namespaces because the preprocessor runs before the compiler ([link](https://stackoverflow.com/a/11791202)).
I specially inserted prefix `SNIPPETS_TESTS_` to avoid possible conflict of definitions in optimized compilation scenario which combine translation units with own `STATIC_SHAPES` macro (if there are) into one unit.

Reply:
Ok, fair enough